viscy/preprocessing/precompute.py

```diff
@@ -0,0 +1,76 @@
+"""Precompute normalization and store a plain C array"""
+
+from pathlib import Path
+from typing import Generator, Literal
+
+import dask.array as da
+from dask.diagnostics import ProgressBar
+from iohub.ngff.nodes import Plate, Position, Well, open_ome_zarr
+
+
+def _normalize_image(
```

Comment:
this is so that we can compute the normalization per timepoint right?

Reply:
In principle yes, but it's currently not used in training.